xsd-fu/templates/cpp/MetadataStore.template

```diff
@@ -250,6 +250,14 @@ namespace ome
         virtual void
         setUUID(const std::string& uuid) = 0;
 
+        /**
+         * Set the Creator associated with this collection of metadata.
+         *
+         * @param creator the Creator to set.
+         */
+        virtual void
+        setCreator(const std::string& creator) = 0;
```

Comment:
The main issue with this change and `getCreator` below is that the addition of methods to the `MetadataRetrieve/MetadataStore` interface is a breaking change. As an example this broke downstream code implementing the `MetadataStore` interface.

Given the OME codebase now requires Java 8 and above, would it be possible to use default methods to workaround this issue and make this change a backwards-compatible API addition?

Reply:
This should only need doing for the Java implementation.  There are no known users of the C++ implementation other than OME Files C++, and that's no longer consuming the version in this repository.